### ISSUE TITLE
(Suggestion) Add event priorities

### ISSUE THREAD

**Issue by MinecraftMan1013number2** [SkriptLang/Skript#3882]
Hello,
I know that in Java, you can add event priorities by doing `@EventHandler(priority = EventPriority.NORMAL)`. I am suggesting that you add something like this in Skript. For example we could do something like
```
on break:
    set event-priority to highest
    # other code
```

I think that this could be very helpful for Skripters trying to override other plugin commands/operations.

**Comment by TheDGOfficial** [comment]
Duplicate of #197. Please use the search feature in Issues section and/or your favorite search engine before submitting new issues, and, next time, fill out the issue template and do not create a blank issue. Your request is implemented in #3852 and you can get a nightly testing JAR (not production ready yet) from here:

https://github.com/SkriptLang/Skript/actions?query=branch%3Afeature%2Fper-event-priority

Click the first commit there you see (currently "Add per-event priority"), then click on skript-nightly on Artifacts section and the download should start. Place the Skript.jar in skript-nightly.zip file to your plugins folder as usual and don't forget to take backups.

Also - the example you provided would be invalid. How you are supposed to change priority of the event listener inside event itself? Like that should change it for next triggered event, etc.? It is better off to be added as like on break with priority highest: (current implementation), or, either adding something like Java Annotations to Skript.

Or maybe you mean something like commands? Like this?
```
on break:
    priority: highest
    trigger:
        # other code
```

**Comment by FranKusmiruk** [comment]
Duplicate of #197
